### PR TITLE
Added Sentiment Analysis using Deep Learning

### LINKED ISSUE #121: Add Sentiment Analysis using Deep Learning
### Description

I want to add the feature of Sentiment Analysis using Deep Learning with better accuracy. I will try to build a better neural network that tries to do sentiment analysis.
Please assign it to me.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Garvit414**
# Fixes Issue🛠️

Closes #121 

# Description👨‍💻 

Dependencies: Tensorflow_hub

I have implemented Sentiment analysis using deep learning and a pre-trained model which has delivered impressive results and black lint the mai.py as well

Target: rating
# Type of change📄

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅
I have run the code on Google Colab and it gives fantastic results.

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷
Screenshot 2023-07-06 234227

### CODE REVIEW

**Comment by Rakesh9100** [comment]
LGTM!! Congrats on getting your PR merged to this repo 🎉 @Garvit414
